tableau/v_tableau.zendesk_tickets.sql

```diff
@@ -1,39 +1,135 @@
 USE gabby
 GO
+--CREATE OR ALTER VIEW tableau.zendesk_tickets AS
 
-CREATE OR ALTER VIEW tableau.zendesk_tickets AS
+WITH original_group AS (
+  SELECT sub.ticket_id
+        ,g.[name] AS group_name
+  FROM
+      (SELECT ticket_id
+             ,[value]
+             ,ROW_NUMBER() OVER(
+              PARTITION BY ticket_id
+              ORDER BY updated) AS group_rn
+              FROM gabby.zendesk.ticket_field_history fh
+              WHERE field_name = 'group_id')
+              AS sub
+  JOIN zendesk.[group] g
+    ON sub.[value] = g.id
+  WHERE sub.group_rn = 1
+ )
+
+,group_updated AS(
+  SELECT ticket_id
+        ,MAX(updated) AS group_updated
+  FROM gabby.zendesk.ticket_field_history fh
+  WHERE field_name = 'group_id'
+  GROUP BY ticket_id
+ )
+
+,original_assignee AS (
+  SELECT sub.ticket_id
+        ,sub.assignee_rn
+        ,sub.updated AS assignee_updated
+        ,u.name AS assignee_name
+        ,scw.primary_on_site_department as orig_assignee_dept
+        ,scw.primary_job AS orig_assignee_job
+        ,scw.legal_entity_name
+  FROM (
+        SELECT *,
+               ROW_NUMBER() OVER(
+               PARTITION BY ticket_id, field_name
+               ORDER BY updated) AS assignee_rn
+          FROM gabby.zendesk.ticket_field_history fh
+        WHERE field_name = 'assignee_id' )
+        AS sub
+  JOIN zendesk.[user] u
+    ON sub.[value] = u.id
+  JOIN gabby.people.staff_crosswalk_static scw
+    ON u.email = scw.userprincipalname
+  WHERE sub.assignee_rn = 1
+ )
+
+,submitter_crosswalk AS (
```

Comment:
don't think you need this, you have user JOINed to below

Reply:
took out the submitter_crosswalk CTE -- were you saying to also take out the sub.assignee_rn=1?

Reply:
nope just the CTE